lib/response.js

```diff
@@ -138,7 +138,7 @@ module.exports = {
     // no content
     if (null == val) {
       if (!statuses.empty[this.status]) this.status = 204;
-      if (val === null) this._explicitNullBody = true;
+      // if (val === null) this._explicitNullBody = true; no need, we can do it at application.js
```

Comment:
You are removing this assignment here with the comment it can be done in Application. but I don't see you setting this in Application?

Reply:
"_explicitNullBody" is used for check if body  is null and used by application.js?
but at line 255 of application.js the code has do the check before. so "_explicitNullBody" is unnecessary.